docs/tasks/tools/install-kubectl.md

```diff
@@ -137,7 +137,7 @@ re-run install-kubectl.ps1 to install latest binaries
        cd C:\users\yourusername (Or wherever your %HOME% directory is)
        mkdir .kube
        cd .kube
-       touch config
+       New-Item config -type file
```

Comment:
I'm not super-confident in this space, but it looks like this should be:

    New-Item -ItemType file config

per the Q&A at https://superuser.com/a/540935/43185

Reply:
![image](https://user-images.githubusercontent.com/19992726/36134192-67f5148e-1083-11e8-9dc5-b2f879cf6630.png)
they are both equivalent commands, I created the two files above using both commands